Sources/MagicExt-OAuth/Core/OAuthExtension.swift

```diff
@@ -122,7 +123,21 @@ public class OAuthExtension: BaseModule {
 }
 
 extension Magic {
+    @available(iOS 14.0, *)
+    private static let logger = Logger(
```

Comment:
Can we make it a utility functions in MagicSDK, so that all extensions can directly call from it and reduce duplicate codes?